Update layout guides panel (#4306)
qgis/QGIS-Documentation: source/docs/user_manual/print_composer/overview_composer.rst

* the |unchecked| :guilabel:`Exclude page from exports` to control whether the
  current page with its content should be included in the :ref:`layout output
  <create-output>`;
* the :guilabel:`Background` of the current page using the :ref:`color

>   <color-selector>` or :ref:`symbol <symbol-selector>` you want. 

Reviewer: ```suggestion
  <color-selector>` or :ref:`symbol <symbol-selector>` you want.
```

page to assist you on items placement, when creating, moving or resizing them.
To be active, guides require the :menuselection:`View --> Show Guides` and
:menuselection:`View --> Snap to Guides` options to be checked.

> To create a guide, different ways:

Reviewer 2: ```suggestion
To create a guide, there are two different methods:
```
?

> * assuming :menuselection:`View --> Show Rulers` option is set, drag out a

Reviewer: ```suggestion
* if the :menuselection:`View --> Show Rulers` option is set, drag out a
```

Reviewer: or "assuming that the"

  ruler and release the mouse button within the page area, at the desired
  position.

> * more accurate, use the :guilabel:`Guides` panel from the :menuselection:`View

Reviewer: ```suggestion
* for more precision, use the :guilabel:`Guides` panel from the :menuselection:`View
```

   The Guides panel

The :guilabel:`Guides` panel allows creation of snap lines at specific
locations:

#. Select the :guilabel:`Page` you'd like to add the guides to
#. Click the |signPlus| :sup:`Add new guide` button and enter the coordinates
   of the horizontal or vertical line. The origin is at the top right corner.

>    Different units are available for setting.

Reviewer: ```suggestion
   Different units are available for this.
```

>    The panel also allows adjusting position of existing guides to exact

Reviewer: ```suggestion
   The panel also allows adjusting the position of existing guides to exact
```

Reviewer: This one should be fixed.  If you want plural, you can skip "the", but have to use "positions".

Author: This one is not the issue! I was not inspired in how i could rephrase the whole sentence to avoid the use of "coordinates" in the next line as well as too many "value"

>    coordinates: double-click a guide coordinate and replace the value.

Reviewer: ```suggestion
   coordinates: double-click and replace the value.
```

Reviewer: What about this?

>    coordinates: double-click a guide coordinate and replace the value.

Reviewer 2: ```suggestion
   coordinates: double-click a guide value and replace the value.
```
? I would prefer to keep the word coordinates for the geographic context.

> #. The :guilabel:`Guides` panel lists only items for the current page.

Reviewer: ```suggestion
#. The :guilabel:`Guides` panel only lists items for the current page.
```

Author: Written this way, i understand that the panel does nothing else than listing, though you can add new items to the page. The limitation is actually on what is listed not on the capabilities of the panel itself imho. But maybe I'va a translation bias
```suggestion
#. The :guilabel:`Guides` panel lists only the items for the current page.
```
?

Reviewer: I am fine with this suggestion.

   It allows creation or removal of guides only in the current page.
   However, you can use the :guilabel:`Apply to All Pages` button to replicate
   the guide configuration of the current page to the other pages in the layout.

> #. To delete a guide, select it and press |signMinus| :sup:`Remove selected

Reviewer: ```suggestion
#. To delete a guide, select it and press the |signMinus| :sup:`Remove selected
```
